Anytype/Sources/PresentationLayer/Flows/EditorPageFlow/EditorPageCoordinatorView.swift

```diff
@@ -46,5 +46,6 @@ struct EditorPageCoordinatorView: View {
                 UndoRedoView(objectId: $0.value)
             }
             .snackbar(toastBarData: $model.toastBarData)
+            .openUrl(url: $model.openUrlData)
```

Comment:
Editor can contains https lins and deeplinks (like anytype://...). `openUrl` can open both links, as legacy url opener